src/tye/Program.DeployCommand.cs

```diff
@@ -29,7 +30,15 @@ public static Command CreateDeployCommand()
                 Required = false
             });
 
-            command.Handler = CommandHandler.Create<IConsole, FileInfo, Verbosity, bool, bool>(async (console, path, verbosity, interactive, force) =>
+            command.AddOption(new Option(new [] { "-n", "--namespace"})
+            {
+                Description = "Specify the namespace for the deployment",
+                Required = false,
+                Argument = new Argument<string>(),
+                Name = "ns"
+            });
+
+            command.Handler = CommandHandler.Create<IConsole, FileInfo, Verbosity, bool, bool, string>(async (console, path, verbosity, interactive, force, @namespace) =>
```

Comment:
Does specifying `ns` instead of `@namespace` work here?

Reply:
`@namespace` is better 😆

Reply:
However you may need to have https://github.com/dotnet/tye/pull/400/files#diff-c1c4c07b0626c051d8c7848f451427c8R38 match the parameter name of the lambda here. This command line parser does something kinda wierd where it requires the parameter names of the lambda to match how the parameters are defined. 

I say if it's working, leave it as-is :)

Reply:
this does work. `ns` doesn't because of the naming.